Answers.md

```diff
@@ -0,0 +1,11 @@
+#1 
+Server-side routing was the way the internet worked when web-browser were not able to process too much information, and will cause the end user to only be given only the data for that specific url. Going to a different url, on the same website, will result in the server fetching all new data to the end user, which was inefficient as there would be a constant flow of data to and from the server. Client-side routing is when the client receives a lot more data, and the browser takes care of the routing by displaying and hiding components as needed when the user changes the url on the same website (changes url by clicking on components of the websites, which is a Link that works together with a Route). So, in modern day, web-browsers have enough processing to handle routing on its own, which results in more efficient data transfer to and from the server.
```

Comment:
This is an awesome answer! It's really important to understand the history of the web, in my opinion, and the struggles of the last era of computing to really make the most out of what tools we have access to now :)